prisma/migrations/20240918030752_add_mark_for_deletion_to_neovim_config/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "NeovimConfig" ADD COLUMN     "markedForDeletion" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
❓ Can we remove this migration completely and just keep the one migration for the index?

Reply:
Done.